src/pages/Users/index.tsx

```diff
@@ -256,14 +262,20 @@ const TableList: React.FC = () => {
                   }
                 }}
               />,
-              <Button
-                loading={loadingExport}
-                type="primary"
-                key="primary"
-                onClick={() => handleDownload()}
+              <Dropdown
```

Comment:
look at this component https://github.com/EscolaLMS/Admin/blob/main/src/components/AuthenticatedLinkButton/index.tsx